substrate/client/rpc-spec-v2/src/chain_head/chain_head.rs

```diff
@@ -110,6 +110,8 @@ pub struct ChainHead<BE: Backend<Block>, Block: BlockT, Client> {
 	/// The maximum number of items reported by the `chainHead_storage` before
 	/// pagination is required.
 	operation_max_storage_items: usize,
+	/// Limit the RPC functionality to a single connection.
```

Comment:
I would re-phrase this, I think you are trying to say that each connection has its own state and not that the RPC functionality is limited to a single connection? 

This connection state is then used to determine whether a certain call is permitted on a particular connection such as "unstable_storage calls" or something like that...

Reply:
I've rephrased the documentation a bit, let me know if it makes more sense now 🙏